.github/workflows/publish-results.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish results
+
+on:
+  workflow_run:
+    workflows: ["Benchmark"]
+    types:
+      - completed
+
+jobs:
+  publish_results:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+      - name: Install Requirements
+        run: pip install -r requirements.txt
+      - name: Copy asv.conf
+        run: cp -f asv.conf.json ci-results/
```

Comment:
I've not quite caught up with why we need to copy this file, but if you need to run asv with a different conf file it seems all the commands have a `--config` option to pass in a specific  configuration file 🤔

https://asv.readthedocs.io/en/stable/commands.html#id4

Reply:
I have made changes to the directory structure such that the `results` are now being stored in the results directory itself and they are published to the `ci-results-preview directory`.